Puzzlepart/did
# Dark theme improvements

### Your checklist for this pull request
 
### Review checklist

### Description
Lightened dark mode hover color, useful for checkboxes and toggles (see #964)


### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Navigate to a page with checkboxes or toggles, e.g. /admin/subscriptionsettings -> Azure AD sync
3. Observe that the hover color is lighter than the background, and that the primary theme color is blue, identical to the light mode blue.

### Related issues
closes #964
